python/QC4_HV.py

```diff
@@ -0,0 +1,24 @@
+from subprocess import CalledProcessError
+from wrappers import runCommand
+from options import parser
+
+args = parser.parse_args()
+cmd = ["python","/Users/anastasia/desktop/QC_Plotting_Tools/Produce_Config_File.py"]
+
+for f in args.file:
+	cmd.append(str(f))
+	pass
+
+cmd.append("--CanvTitleX=Divider Current #left(#muA#right)")
+cmd.append("--CanvTitleY=Applied Voltage #left(kV#right)")
+cmd.append("--LatexLine2=0.62,0.20, R_{Equiv}~=~5.000~M#Omega")
```

Comment:
The value of `R_{Equiv}` here should not be hard coded.  The user should be measuring this with a multimeter at the time of the experiment (not assume it is 5.0) and then use this value here.  This should be taken from the xls sheet and plugged in here.